pool/connection_pool.go

```diff
@@ -1417,7 +1425,10 @@ func (p *ConnectionPool) controller(ctx context.Context, e *endpoint) {
 					// Relocate connection between subpools
 					// if ro/rw was updated.
 					if e.conn == nil {
-						p.tryConnect(ctx, e)
+						if err = p.tryConnect(ctx, e); err != nil {
+							log.Printf("tarantool: reopen connection to %s failed: %s\n",
+								e.name, err)
+						}
```

Comment:
The method-level scope of the variable is confusing a bit. The local scope is enough.
```suggestion
						if err := p.tryConnect(ctx, e); err != nil {
							log.Printf("tarantool: reopen connection to %s failed: %s\n",
								e.name, err)
						}
```

Reply:
done